ors-test-scenarios/pom.xml

```diff
@@ -16,6 +16,7 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <surefireArgLine></surefireArgLine>
```

Comment:
`<surefireArgLine>` is filled by the parent pom, and seems to be necessary as an empty argument here to not run into issues.